mono/metadata/icall.c

```diff
@@ -8291,7 +8291,7 @@ no_icall_table (void)
  * If the method is not found, warns and returns NULL.
  */
 gpointer
-mono_lookup_internal_call_full (MonoMethod *method, mono_bool *uses_handles)
+mono_lookup_internal_call_full2 (MonoMethod *method, gboolean warn_on_missing, mono_bool *uses_handles)
```

Comment:
Since `mono_lookup_internal_call_full` is now dead-code (it's not `MONO_API` and nothing else use it in the runtime AFAICT), why not simply adding the parameter to `mono_lookup_internal_call_full`?